.github/workflows/dsf-dam_poc_cli.yml

```diff
@@ -1,4 +1,4 @@
-name: 'DSF & DAM POC CLI'
+name: 'DSF POC CLI - ALL'
```

Comment:
Adding "All" is not aligned with the convention Assaf and I put in place. What does it mean? No other yml has All

Reply:
This file should be renamed to dsf_poc_cli.yml before the next release

Reply:
Please see again, I removed dsf-dam_poc_cli.yml and used the dsf_poc_cli.yml for it.

Reply:
Great, thanks